cuda_bindings/cuda/bindings/_bindings/loader.cpp

```diff
@@ -1,11 +1,6 @@
 // Copyright 2021-2024 NVIDIA Corporation.  All rights reserved.
-//
-// Please refer to the NVIDIA end user license agreement (EULA) associated
-// with this source code for terms and conditions that govern your use of
-// this software. Any use, reproduction, disclosure, or distribution of
-// this software and related documentation outside the terms of the EULA
-// is strictly prohibited.
-#include <stdio.h>
```

Comment:
It seems the include is gone. (Also, we're using `<stdio.h>` instead of `<cstdio> in a C++ file...? 🤔 ditto for stdlib below)

Reply:
Fixed: commit 85e0182fa984188c6685fd5164e40eb2c5eb4483

Thanks for catching this accident! (I was going through too fast with a vim macro.)